### DISCUSSION TITLE
连续测RFC3489结果会从nat3变nat1

### DISCUSSION THREAD

**Discussion by yxok200** [HMBSbige/NatTypeTester#254] [Q&A]
![image](https://github.com/user-attachments/assets/35624c0d-8c16-44aa-95f2-41a7c660e8df)
首次测试结果是PortRestrictedCone，端口受限型NAT3

![image](https://github.com/user-attachments/assets/f72b411c-40bb-4115-811a-5a6a5632e7c9)
段时间内再次点测试，会出来FullCone的结果

**Comment by HMBSbige** [comment]
Duplicate of #4
